osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -73,35 +99,47 @@ public override double Calculate(Dictionary<string, double> categoryRatings = nu
             return totalValue;
         }
 
-        private double computeAimValue()
+        /// <summary>
+        /// Calculates achieved star rating for current score
+        /// </summary>
+        /// <param name="comboDifficulties">Array of map star ratings</param>
+        /// <returns>Current score star rating</returns>
```

Comment:
Needs better xmldoc/explanation, especially in how this differs from `missCountStarRating`.